R/class-workbook.R

```diff
@@ -4185,9 +4282,8 @@ wbWorkbook <- R6::R6Class(
       self
     },
 
-    ws = function(sheet) {
-      sheet_id <- wb_validate_sheet(self, sheet)
-      self$worksheets[[sheet_id]]
+    get_worksheet = function(sheet) {
+      self$worksheets[[private$get_sheet_index(sheet)]]
```

Comment:
Shall we simply remove this? It never worked the way I intended it to be and now I no longer care ...

Reply:
I think as we refactor some code we'll find a use for it as a nice shortcut